k8s/dev/logservice.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
```

Comment:
note: probably not wanted in dev/. dev/ is meant for deployments we only use while developing (like the postgres service)

Reply:
maybe we should rename it, but the /dev is use by tilt

Reply:
woops! my bad - i confused this for test/ - I didn't realize we added dev/.

Reply:
Should we add these to deployments/ too then?

Reply:
sorry I was confused? This is already in dev and we don't want to add this to deployment because we probably will do this differently?

Reply:
https://github.com/chroma-core/chroma/blob/main/k8s/deployment/kubernetes.yaml we should add the deployment for the log service in that file too.

Reply:
oh I see, this is what is used for minikube and kube apply. ok I can add that. Can we make them into one version later?